test-support/src/main/java/org/springframework/ldap/test/TestContextSourceFactoryBean.java

```diff
@@ -94,6 +115,10 @@ public void setContextSource(ContextSource contextSource) {
         this.contextSource = contextSource;
     }
 
+	public void setAdditionalSchemas(List<String> additionalSchemas) {
```

Comment:
Add some Javadoc which includes examples on common values that someone might want to add.